cognite/client/data_classes/shared.py

```diff
@@ -34,8 +34,6 @@ def __init__(self, count: int = None, **kwargs: Any):
         self.count = count
         self.update(kwargs)
 
-    count = CognitePropertyClassUtil.declare_property("count")
```

Comment:
Good catch. The `declare_property` util is a remnant from when we autogenerated data classes, so I'd like to get rid of those. But we should pass the count argument to the dict's constructor, so that the value ends up in there too.

Reply:
and no idea why the API returns both value and values.. I'll create a ticket on that too. But these aggregation endpoints will be replaced soon anyway, so I don't really think any of this will be fixed.